modules/sticky-messages/events/messageCreate.js

```diff
@@ -3,37 +3,38 @@ const channelData = {};
 
 /**
  * Deletes the sticky message sent by the bot
- * @param {Discord.Client} client
- * @param {Discord.Message} msg
+ * @param {Snowflake} clientId User ID of the bot
+ * @param {Discord.TextBasedChannel} channel
  */
-async function deleteMessage(client, msg) {
+async function deleteMessage(clientId, channel) {
     let message;
-    message = await msg.channel.messages.fetch(channelData[msg.channel.id].msg).catch(async () => {
-        const msgs = await msg.channel.messages.fetch({limit: 20});
-        message = msgs.find(m => m.author.id === client.user.id);
+    message = await channel.messages.fetch(channelData[channel.id].msg).catch(async () => {
```

Comment:
`channelData[channel.id]` might be `undefined` if the module configuration gets changed and reloaded before messages where sent in the configured channel, leading to a TypeError